apache/incubator-hawq
# HAWQ-958, HAWQ-976, HAWQ-957. LICENSE & NOTICE updates

* Update Stream license
* Update PyGreSQL license
* Update CMake license
* Remove ASF license header from CMake files
* Update pom.xml to exclude all libyarn and libhdfs3 CMake files
* Using new license format seen in spark.
* HAWQ-976. dynloader license (and corresponding source change)
* HAWQ-976. glob license (and corresponding source change)
* Add unittest2, pychecker, pg_controldata, pg8000, lockfile LICENSE files
* Add figleaf LICENSE file
* Remove incorrect ASF headers for figleaf
* update bzip2 license
* update googletest license
* update pexpect license
* update pitlockfile license
* update pljava license
* update oraface license
* Add licenses for:
    LICENSE-json.txt
    LICENSE-plperl.txt
    LICENSE-postgresql.txt
    LICENSE-test-ctype.txt
* Fix incorrect GPL reference - tools/bin/gppylib/operations/test/test_package.py
* HAWQ-957, HAWQ-952. Update NOTICE & remove COPYRIGHT
* Remove ASF license header for plperl
* Add PSI,regents-of-the-university-of-california, university-of-california-at-berkeley LICENSE file
* Add port, wstrcmp BSD license
* Add isc, libpq-sha2, port-rand, port-gettimeofday, regex license
* Add pg_dump license
